### ISSUE TITLE
Sybase strore procedures fragmented in DBeaver 5.3.2

### ISSUE THREAD

**Issue by LMunozO** [dbeaver/dbeaver#5002]
#### System information: 
- Operating system (distribution) and versión Win 10 64 Bits enterprise
- DBeaver versión 5.3.2
- Java version 
- Additional extensions

#### Connection specification:
- Database name and versión Sybase 16
- Driver name jTDS
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
When opening the Procedures Branch all the procedures appears as repeated, but they are not repeated, every name contains a fragment of the store procedure

#### Steps to reproduce, if exist:
Connect to the database
open dbo schema
open Procedures Branch to see the list of procedures
#### Include any warning/errors/backtraces from the logs

2019-01-14 12:46:54.573 - DBeaver 5.3.2.201901062110 is starting
2019-01-14 12:46:54.589 - OS: Windows 10 10.0 (amd64)
2019-01-14 12:46:54.589 - Java version: 1.8.0_77 by Oracle Corporation (64bit)
2019-01-14 12:46:54.589 - Install path: 'C:\Users\LMUNOZ\AppData\Local\DBeaver'
2019-01-14 12:46:54.589 - Instance path: 'file:/C:/Users/LMUNOZ/.dbeaver4/'
2019-01-14 12:46:54.589 - Memory available 61Mb/910Mb
2019-01-14 12:46:54.604 - Starting RMI server at 57956
2019-01-14 12:46:54.713 - Run workbench
2019-01-14 12:46:54.713 - > Start org.eclipse.e4.ui.workbench.swt [0.14.300.v20180906-1121]
2019-01-14 12:46:54.713 - > Start org.eclipse.e4.core.contexts [1.7.100.v20180817-1215]
2019-01-14 12:46:54.729 - > Start org.eclipse.e4.core.services [2.1.200.v20180817-1215]
2019-01-14 12:46:54.729 - > Start org.eclipse.e4.ui.services [1.3.200.v20180906-1121]
2019-01-14 12:46:54.792 - > Start org.eclipse.e4.ui.di [1.2.300.v20180906-1121]
2019-01-14 12:46:54.870 - > Start com.ibm.icu.base [58.2.0.v20170418-1837]
2019-01-14 12:46:54.870 - > Start org.eclipse.emf.common [2.15.0.v20180723-1316]
2019-01-14 12:46:54.885 - > Start org.eclipse.emf.ecore [2.15.0.v20180722-1159]
2019-01-14 12:46:54.932 - > Start org.eclipse.emf.ecore.xmi [2.15.0.v20180706-1146]
2019-01-14 12:46:55.120 - > Start org.eclipse.core.filesystem [1.7.200.v20180828-0158]
2019-01-14 12:46:55.354 - > Start org.eclipse.core.resources [3.13.100.v20180828-0158]
2019-01-14 12:46:55.479 - > Start org.eclipse.e4.ui.workbench.addons.swt [1.3.200.v20180828-0227]
2019-01-14 12:46:55.479 - > Start org.eclipse.core.expressions [3.6.200.v20180817-1401]
2019-01-14 12:46:55.495 - > Start org.eclipse.e4.ui.workbench.renderers.swt [0.14.300.v20180828-0227]
2019-01-14 12:46:55.604 - > Start org.eclipse.e4.ui.bindings [0.12.200.v20180828-0227]
2019-01-14 12:46:55.651 - > Start org.eclipse.e4.core.di.extensions [0.15.200.v20180817-1215]
2019-01-14 12:46:55.667 - > Start org.eclipse.ui.ide [3.14.100.v20180828-1350]
2019-01-14 12:46:55.932 - > Start org.eclipse.e4.ui.css.swt [0.13.200.v20180828-0227]
2019-01-14 12:46:55.982 - > Start org.eclipse.e4.ui.css.swt.theme [0.12.100.v20180828-0227]
2019-01-14 12:46:56.560 - Initializing DBeaver 5.3.2
2019-01-14 12:46:56.560 - Host plugin: org.jkiss.dbeaver.core.application 5.3.2.201901062110
2019-01-14 12:46:56.560 - Initialize Core...
2019-01-14 12:46:56.575 - Can't find BC security provider. Use default JCE.
2019-01-14 12:46:56.575 - BouncyCastle not registered, using the default JCE provider
2019-01-14 12:46:56.747 - > Start org.eclipse.team.core [3.8.300.v20180827-1041]
2019-01-14 12:46:56.763 - > Start org.jkiss.dbeaver.ui.navigator [1.0.0.201901062110]
2019-01-14 12:46:56.825 - Total database drivers: 68 (68)
2019-01-14 12:46:56.888 - > Start org.eclipse.equinox.security [1.2.500.v20180827-1235]
2019-01-14 12:46:56.888 - > Start org.eclipse.ui.net [1.3.300.v20180827-1041]
2019-01-14 12:46:56.904 - Proxy service 'org.eclipse.core.net.proxy.IProxyService' loaded
2019-01-14 12:46:56.904 - > Start org.eclipse.core.net [1.3.300.v20180827-1041]
2019-01-14 12:46:56.904 - Core initialized (344ms)
2019-01-14 12:46:56.904 - Configure workbench window
2019-01-14 12:46:56.919 - > Start org.eclipse.help [3.8.200.v20180821-0700]
2019-01-14 12:46:57.982 - Create actions
2019-01-14 12:46:57.982 - Disable Eclipse action set 'org.eclipse.ui.edit.text.actionSet.annotationNavigation'
2019-01-14 12:46:57.997 - Disable Eclipse action set 'org.eclipse.ui.WorkingSetActionSet'
2019-01-14 12:46:58.029 - > Start org.eclipse.ui.editors [3.11.200.v20180822-1511]
2019-01-14 12:46:58.029 - > Start org.eclipse.core.filebuffers [3.6.300.v20180822-1339]
2019-01-14 12:46:59.060 - > Start org.eclipse.search [3.11.300.v20180822-1339]
2019-01-14 12:46:59.153 - > Start org.eclipse.ui.workbench.texteditor [3.11.100.v20180824-1030]
2019-01-14 12:46:59.825 - Initialize workbench window
2019-01-14 12:46:59.825 - Finish initialization
2019-01-14 12:46:59.935 - > Start org.jkiss.dbeaver.ui.editors.base [1.0.0.201901062110]
2019-01-14 12:47:00.700 - > Start org.jkiss.dbeaver.ui.editors.data [1.0.0.201901062110]
2019-01-14 12:47:00.966 - > Start org.eclipse.equinox.p2.core [2.5.100.v20180822-1532]
2019-01-14 12:47:00.981 - > Start org.eclipse.equinox.p2.operations [2.5.100.v20180822-1302]
2019-01-14 12:47:00.981 - > Start org.eclipse.equinox.p2.engine [2.6.100.v20180822-1302]
2019-01-14 12:47:00.981 - > Start org.eclipse.equinox.p2.metadata [2.4.100.v20180822-1302]
2019-01-14 12:47:00.997 - > Start org.eclipse.equinox.p2.ui.sdk.scheduler [1.4.100.v20180827-1122]
2019-01-14 12:47:01.013 - > Start org.eclipse.equinox.p2.updatechecker [1.2.100.v20180827-1122]
2019-01-14 12:47:01.013 - > Start org.eclipse.equinox.p2.metadata.repository [1.3.100.v20180822-1302]
2019-01-14 12:47:01.013 - > Start org.eclipse.equinox.p2.repository [2.4.100.v20180822-1354]
2019-01-14 12:47:01.263 - > Start org.jkiss.dbeaver.ext.sample.database [1.0.37.201901062110]
2019-01-14 12:47:01.263 - > Start org.jkiss.dbeaver.ext.ui.tipoftheday [1.0.6.201901062110]
2019-01-14 12:47:01.310 - > Start org.eclipse.equinox.p2.ui [2.5.200.v20180822-1357]
2019-01-14 12:47:02.841 - > Start org.jkiss.dbeaver.ext.erd [2.0.102.201901062110]
2019-01-14 12:47:02.856 - > Start org.jkiss.dbeaver.ui.editors.sql [1.0.0.201901062110]
2019-01-14 12:47:07.262 - Connect with 'invex_ci' (sybase_jtds-1684d6346ae-491ac6e999ff951a)
2019-01-14 12:47:07.278 - > Start org.jkiss.dbeaver.ext.mssql [1.0.94.201901062110]
2019-01-14 12:47:07.340 - > Start org.jkiss.dbeaver.ext.generic [2.3.86.201901062110]
2019-01-14 12:47:07.747 - Connected (sybase_jtds-1684d6346ae-491ac6e999ff951a, jTDS Type 4 JDBC Driver for MS SQL Server and Sybase [1.3.1])
2019-01-14 12:47:09.215 - Can't get column 'schema_id': Invalid column name schema_id.

**Comment by serge-rider** [comment]
In ticket #4378 we fixed the query which read procedure list.
Now we use this one:
```sql
select distinct so.name as proc_name,su.name as schema_name,so.loginame as login_name,sc.text as definition
from syscomments sc, sysobjects so, sysusers su
where so.id = sc.id 
and so.type = 'P'
and su.uid = so.uid
and su.name=SCHEMA_NAME
```
Perhaps we need to use GROUP BY to filter duplicates..

**Comment by serge-rider** [comment]
Ah, I found the bug. We don't need to join with `syscomments`.
Thanks for the bug report.

**Comment by lozbrown** [comment]
I came to test the fix for #4378 and found the same issue. Unfortunately the above suggests that my suggested SQL seems to have been the cause. @serge-rider  Apologies for that.
